changelog/new_namingfilename_make.md

```diff
@@ -0,0 +1 @@
+* [#10220](https://github.com/rubocop/rubocop/issues/10220): Naming/FileName: make CheckDefinitionPathHierarchy roots configurable. ([@grosser][])
```

Comment:
```suggestion
* [#10220](https://github.com/rubocop/rubocop/issues/10220): Update `Naming/FileName` to make `CheckDefinitionPathHierarchy` roots configurable. ([@grosser][])
```

To match our general changelog style.

Reply:
This changelog should use `change` instead of `new` as well.

Reply:
done